### PR TITLE
Visually match the front-facing portion of site.

### PR DESCRIPTION
**Pull request by victorx716**
Uses flexbox on flex items as well as the bigger components. Refactors the spacing to match the item detail page.

### CODE REVIEW

**Comment by victorx716** [comment]
screen shot 2019-02-15 at 12 06 43 pm

File: client/src/components/Sidebar/Feature/Feature.jsx [hunk 1]
```diff
@@ -1,20 +1,78 @@
...
+background-color: transparent;
+outline: none;
+text-align: left;
+`;
+
+const Rating = styled.div`
+  color: #797979;
+  display: inline-block;
+`;
+
```
Comment (pafrias) [inline client/src/components/Sidebar/Feature/Feature.jsx:35]: I'd suggest new title for keep, can't really tell what it means on the page.

**Comment by pafrias** [comment]
Overall the styled components are very good, only suggestion I'd make would be that the file names could be a little clearer as to what element on the page they refer to.

File: client/src/components/Sidebar/Feature/Feature.jsx (hunk 1 above)
Comment (victorx716) [reply client/src/components/Sidebar/Feature/Feature.jsx:35]: Gotcha. Just modified this.

Comment (victorx716) [reply client/src/components/Sidebar/Feature/Feature.jsx:35]: I'll modify the other ones as well.

**Pull request merged**